scripts/aad-app-reg.sh

```diff
@@ -404,16 +428,30 @@ if [[ -n ${apiAppObjectId} ]]; then
     apiAppId=$(az ad app show --id ${apiAppObjectId} --query "appId" --output tsv)
     echo "API app registration with ID ${apiAppId} updated"
 else
+    echo "Creating a new API app registration, ${appName} API"
     apiAppId=$(az rest --method POST --uri "${msGraphUri}/applications" --headers Content-Type=application/json --body "${apiApp}" --output tsv --query "appId")
-    echo "Creating a new API app registration, ${appName} API, with ID ${apiAppId}"
+    echo "AppId: ${apiAppId}"
 
     # Poll until the app registration is found in the listing.
+    echo "Waiting for the new app registration"
     wait_for_new_app_registration $apiAppId
 
     # Update to set the identifier URI.
+    echo "Updating identifier URI"
     az ad app update --id ${apiAppId} --identifier-uris "api://${apiAppId}"
 fi
 
+echo "Setting API permissions (email / profile / ipaddr)"
+# unfortunatly we need to assign these permissions using the non descriptive guids...(taken from the manifest)
+az ad app permission add --id ${apiAppId} --api 00000003-0000-0000-c000-000000000000 --api-permissions 64a6cdd6-aab1-4aaf-94b8-3cc8405e90d0=Scope 37f7f235-527c-4136-accd-4a02d197296e=Scope 14dad69e-099b-42c9-810b-d002981feec1=Scope
```

Comment:
nit - could we create variables in the script for these GUIDs? At least that way we'd have something that describes them in the script 😄